go/mysql/flavor_mariadb.go

```diff
@@ -154,6 +155,7 @@ func (mariadbFlavor) status(c *Conn) (SlaveStatus, error) {
 	if err != nil {
 		return SlaveStatus{}, vterrors.Wrapf(err, "SlaveStatus can't parse MariaDB GTID (Gtid_Slave_Pos: %#v)", resultMap["Gtid_Slave_Pos"])
 	}
+
```

Comment:
Can you avoid adding whitespace diffs?

Reply:
Yeah, absolutely. I had initially implemented the logic to fill in the new fields in each flavor, and then realized that it's the same for each flavor so I moved that into the shared function all flavors use for filling common fields.

I should have been more diligent about checking diffs, and just checked this file back out from master. I'll do that now and push back up.